documentation/modules/metrics/proc-metrics-kafka-deploy-options.adoc

```diff
@@ -5,45 +5,108 @@
+* `kafka-metrics.yaml`
+* `kafka-connect-metrics.yaml`
+* `kafka-mirror-maker-2-metrics.yaml`
+* `kafka-bridge-metrics.yaml`
+* `kafka-cruise-control-metrics.yaml`
 
-[id='proc-metrics-kafka-{context}']
-== Copying Prometheus metrics configuration to a custom resource
+The example configuration files have relabeling rules and the configuration required to enable Prometheus metrics.
+Prometheus scrapes metrics from target HTTP endpoints.
+The example files are a good way to try Prometheus with Strimzi.
 
-To use Grafana dashboards for monitoring, copy xref:ref-metrics-yaml-files-{context}[the example metrics configuration to a custom resource].
+To apply the relabeling rules and metrics configuration, do one of the following:
 
-In this procedure, the `Kafka` resource is updated, but the procedure is the same for all components that support monitoring.
+* Copy the example configuration to your own custom resource definition
+* Deploy the custom resource with the metrics configuration
 
-.Procedure
-Perform the following steps for each `Kafka` resource in your deployment.
-
-. Update the `Kafka` resource in an editor.
-+
-[source,shell,subs="+quotes,attributes"]
-----
-kubectl edit kafka _KAFKA-CONFIG-FILE_
-----
-
-. Copy the xref:ref-metrics-yaml-files-{context}[example configuration in `kafka-metrics.yaml`] to your own `Kafka` resource definition.
-. Save the file, and wait for the updated resource to be reconciled.
-
-[id='proc-metrics-deploying-kafka-{context}']
-== Deploying a Kafka cluster with Prometheus metrics configuration
+If you want to include Kafka Exporter metrics, add `kafkaExporter` configuration to your `Kafka` resource.
 
-To use Grafana dashboards for monitoring, you can deploy xref:ref-metrics-config-files-{context}[an example Kafka cluster with metrics configuration].
+IMPORTANT: For Kafka Exporter to be able to provide additional Kafka metrics, you also need the Prometheus metrics configuration for Kafka.
 
-In this procedure, The `kafka-metrics.yaml` file is used for the `Kafka` resource.
+This procedure shows how to deploy Prometheus metrics configuration in the `Kafka` resource.
+The process is the same when using the example files for other resources.
 
 .Procedure
 
-* Deploy the Kafka cluster with the xref:ref-metrics-yaml-files-{context}[example metrics configuration].
+* Deploy the example custom resource with the Prometheus configuration.
 +
+For example, for each `Kafka` resource, you would apply the `kafka-metrics.yaml` file.
++
+.Deploying the example configuration
 [source,shell,subs="+attributes"]
 ----
 kubectl apply -f kafka-metrics.yaml
 ----
++
+Alternatively, you can copy the example configuration in `kafka-metrics.yaml` to your own `Kafka` resource definition.
++
+.Copying the example configuration
+[source,shell,subs="+quotes,attributes"]
+----
+kubectl edit kafka _<kafka-configuration-file>_
+----
++
+To deploy Kafka Exporter, add `kafkaExporter` configuration to the `Kafka` resource.
```

Comment:
Since we say that _The process is the same when using the example files for other resources._ ... should we mention here that Kafka Exporter is only in the `Kafka` CR?

Reply:
Okay. Added: _`kafkaExporter` configuration is only specified in the `Kafka` resource._